### ISSUE TITLE
Url contains the port number when trying to access site

### ISSUE THREAD

**Issue by taylorSando** [progrium/dokku#1639]
The problem is that when I try to access the site, it is forwarding the request to the SSL port and showing it in the url.  I've deployed other sites before without this problem.  The last one was dokku 4.2, and the site I'm currently using is dokku 4.3.  No idea if it makes a difference.  The site is hosted on digital ocean.  This is the config in /home/dokku/client/nginx.config

```
server {
  listen      [::]:80;
  listen      80;
  server_name client.app.takemyspot.ca;
  access_log  /var/log/nginx/client-access.log;
  error_log   /var/log/nginx/client-error.log;
  return 301 https://$host:20424$request_uri;
}

server {
  listen      [::]:20424 ssl spdy;
  listen      20424 ssl spdy;
  server_name client.app.takemyspot.ca;
  access_log  /var/log/nginx/client-access.log;
  error_log   /var/log/nginx/client-error.log;
ssl_certificate     /home/dokku/client/tls/server.crt;
ssl_certificate_key /home/dokku/client/tls/server.key;

  keepalive_timeout   70;
  add_header          Alternate-Protocol  20424:npn-spdy/2;
  location    / {

    gzip on;
    gzip_min_length  1100;
    gzip_buffers  4 32k;
    gzip_types    text/css text/javascript text/xml text/plain text/x-component application/javascript application/x-javascript application/json application/xml  application/rss+xml font/truetype application/x-font-ttf font/opentype application/vnd.ms-fontobject image/svg+xml;
    gzip_vary on;
    gzip_comp_level  6;

    proxy_pass  http://client;
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection upgrade;
    proxy_set_header Host $http_host;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header X-Forwarded-For $remote_addr;
    proxy_set_header X-Forwarded-Port $server_port;
    proxy_set_header X-Request-Start $msec;
  }
  include /home/dokku/client/nginx.conf.d/*.conf;
}
upstream client { 
  server 172.17.0.161:5000;
}
```
Everything is getting forwarded to the SSL's port.  I have another app running on the same server, the api, which is running fine and is showing the url without the port.

**Comment by michaelshobbs** [comment]
The port is shown because it's not the default SSL port of 443. This is standard browser behavior as far as I know. Did you intend for the SSL listener to be on a non-standard port?

**Comment by taylorSando** [comment]
I was just using the config that dokku had created.  On the other site using SSL, it is also split into a client and api.  One is using port 443, and the other is using a different port.  The one using the different port doesn't seem to cause forwarding.  I don't really care what ports they are using, as long as they are using SSL, and not letting the end user see a port in the url.

**Comment by michaelshobbs** [comment]
You'll only be able to hide the SSL port in a browser if it's running on the standard port of 443. To troubleshoot further you'll need to destroy the app exhibiting the issue and deploying again with dokku trace on. Check out our contributing doc and troubleshooting doc for more details on what info to include in an issue and how to turn on dokku trace.

**Comment by taylorSando** [comment]
I recreated it, and this time it seemed to set the SSL port to 443.  Maybe it was the order I was creating the app? 

This is what I did this time:
dokku apps:create client
dokku domains:add client.app.takemyspot.ca
dokku certs:add client < /home/cert-key.tar
dokku config:set <My environment variables>

Then locally I did:
git push <app-name>

I know before I tried to enable nginx and do build-config, which I don't think was necessary.
